### DISCUSSION TITLE
Right way to Sharing Docker image between multiple lambda by specifying seperate [cmd] for lambda

### DISCUSSION THREAD

**Discussion by anwarjaved** [aws/aws-cdk#16894] [Q&A]
Hi,
 I want to know what uis the best way to deploy .net 5 docker image and sharing same image between multiple lambdas I also want to upload docker image to specific folder

**Comment by peterwoodworth** [comment]
You can upload an image with [DockerImageAsset](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ecr_assets.DockerImageAsset.html) and then reference that image when creating a Lambda Function with [fromEcrImage](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda.Code.html#static-fromwbrecrwbrimagerepository-props)
